bincommon/commonflags.go

```diff
@@ -112,6 +112,8 @@ var (
 		"Range `min:max` for the max number of connections to reuse for each thread, default to unlimited. "+
 			"e.g. 10:30 means randomly choose a max connection reuse threshold between 10 and 30 requests.").
 		WithValidator(ConnectionReuseRangeValidator(&httpOpts))
+	// NoResolveConnFlag is false if we want to resolve the DNS name for each new connection.
+	NoResolveConnFlag = flag.Bool("no-resolve", true, "Re-resolve the DNS name for each connection")
```

Comment:
you meant false for default right, as we want to turn the new behavior (to re resolve) on

also name would be (despite stutter) `NoReResolveFlag` ? (`-no-reresolve`)

"Keep the initial DNS resolution and don't re-resolve when making new connections (because of error or reuse limit reached)"

Reply:
updated